.github/issue_template/feature.md

```diff
@@ -0,0 +1,13 @@
+---
+name: 🚀 Feature
+about: Submit a proposal for a new feature
```

Comment:
```suggestion
about: Submit a proposal for a new mypy feature
```
Also, I wonder if we should add somewhere that this is *not* the place to propose new type system features -- those should generally be discussed on `typing-sig@python.org` or in https://github.com/python/typing/issues and probably have to go through the PEP process (see https://www.python.org/dev/peps/pep-0001/).